lib/cuke_linter/helpers/dialect_helper.rb

```diff
@@ -0,0 +1,25 @@
+module CukeLinter
+  module DialectHelper
+
+    def self.set_dialect(dialect)
+      @dialect = dialect
+    end
+
+    def self.given_keyword
+      @dialect['given'][1].strip
+    end
+
+    def self.when_keyword
+      @dialect['when'][1].strip
+    end
+
+    def self.then_keyword
+      @dialect['then'][1].strip
+    end
+
+    def self.get_model_dialect(model)
+      set_dialect(Gherkin::DIALECTS[model.parent_model.parsing_data[:language]])
```

Comment:
I'm still averse to having a direct dependency on `gherkin`. Granted, `cuke_modeler` hasn't been updated to expose the language or its keywords yet but it's not like this feature has to go out the door ASAP (if time is an issue for you, I'm confident that you are more than capable of monkey patching a linter or two on your end until a proper release gets out).